src/ansys/fluent/core/solver/flobject.py

```diff
@@ -1208,7 +1210,7 @@ def get_cls(name, info, parent=None, version=None):
             "user_creatable", False
         )
 
-        if version == "222":
```

Comment:
@ansaboutin Because flobject.py is used in Fluent as well, we would need to bring the FluentVersion class into Fluent's codebase. In doing so, we need to keep the imports as simple as possible. So for PyFluent, we can put a proxy module in the same folder as flobject.py to handle the modue import.

Reply:
I think this change should be reverted: it is minor, and this is going to make things unnecessarily more complicated. Leaving `flobject.py` unchanged seems better.